doc/UltiSnips.txt

```diff
@@ -981,6 +1034,37 @@ endsnippet
 wow<tab>Hello World ->
 Hello World                                                     HELLO WORLD
 
+
+The following snippet uses the file store to remember the last test suite used
+
+------------------- SNIP -------------------
+snippet test
+TEST(${1:`!psnip.rv=snip.store.file['test_suite', '']`}, $2){
+    $0
+}`!p
+snip.store.file.test_suite = t[1]
+`
+endsnippet
+
+
+The following snippet implements a counter. It uses the 'tmp' store to make
+sure the counter is incremented only once (else, the snippet update would run the
+code again)
+
+------------------- SNIP -------------------
+snippet inc "increment"
+`!p
+if snip.store.tmp['todo', True]:
+    snip.rv = snip.store.file['i', 1]
+    snip.store.file.i += 1
+    snip.store.tmp.todo = False
```

Comment:
I disagree with this shortcut, let's keep the thing a dictionary and not get fancy with accessors.